crates/node-bindings/src/parcel/parcel.rs

```diff
@@ -25,30 +37,86 @@ impl ParcelNapi {
     let thread_id = std::thread::current().id();
     tracing::trace!(?thread_id, "parcel-napi initialize");
 
+    // Parcel Core Options
+    let mut parcel_options = ParcelOptions::default();
+
     // Wrap the JavaScript-supplied FileSystem
-    let fs = FileSystemNapi::from_options(&env, &options)?;
+    parcel_options.fs = FileSystemNapi::from_options(&env, &options)?;
+
+    // Assign Rust thread count from JavaScript
+    let js_threads = options.get_named_property::<JsUnknown>("threads")?;
+    parcel_options.threads = match js_threads.get_type()? {
```

Comment:
Does this need to be configurable? Can we just use rayon's default?

Reply:
In the past I have found it helpful to control the number of threads for debugging and benchmarking. You can pass this number to Rayon, Tokio when configuring them. I think it's fine to have?